src/FunctionalExtras.Tests/Data/TuplesTests.cs

```diff
@@ -0,0 +1,25 @@
+using System;
```

Comment:
It looks like this is testing `ValueTuple` instances, not `Tuple` instances.

Reply:
Oop. Forgot to rename the file… One moment…

Reply:
Fixed in `e68fded`.